.github/CODEOWNERS

```diff
@@ -329,14 +329,14 @@
 /pkg/collector/corechecks/system/winkmem/         @DataDog/windows-agent
 /pkg/collector/corechecks/system/winproc/         @DataDog/windows-agent
 /pkg/collector/corechecks/systemd/      @DataDog/agent-integrations
-/pkg/collector/corechecks/nvidia/       @DataDog/agent-platform
+/pkg/collector/corechecks/nvidia/       @DataDog/agent-ci-experience
```

Comment:
Note: to be updated later, it doesn't make sense for this to be owned by any of the ex-Agent Platform teams.